### ISSUE TITLE
private_claims still has prefix

### ISSUE THREAD

**Issue by beruhan** [NovemberOscar/Sanic-JWT-Extended#24]
from the doc 	about Storing Private Claims
```
propagated Token object contains private claims in Token.private_claims. prefix is not exist on this time.
```
It seems also prefix still exists

**Comment by rscarrera27** [comment]
OK, I'll find why and fix it in few hours. really thanks for report 🙏

**Comment by rscarrera27** [comment]
@beruhan #23 and #24 are fixed in `1.0.dev11` please upgrade your dependency.

Thanks for your report 🙏
